playbooks/besman-counterfit-0.0.1-playbook.sh

```diff
@@ -4,10 +4,10 @@ function __besman_init() {
     __besman_echo_white "initializing"
     export ASSESSMENT_TOOL_NAME="counterfit"
     export ASSESSMENT_TOOL_TYPE="dast"
-    export ASSESSMENT_TOOL_VERSION="0.1.1"
-    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_NAME-0.0.1-playbook.sh"
+    export ASSESSMENT_TOOL_VERSION="0.0.1"
+    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_NAME-$ASSESSMENT_TOOL_VERSION-playbook.sh"
```

Comment:
This should be besman-$ASSESSMENT_TOOL_NAME-0.0.1-playbook.sh. Because the playbook version can be different from the tool version.

Reply:
Got it.